specs/protocol/isthmus/exec-engine.md

```diff
@@ -127,6 +128,17 @@ In response to RPC methods like `eth_simulateV1` that allow simulation of arbitr
 an empty withdrawals root should be included in the header of a block that consists of such simulated transactions. The same
 is applicable for scenarios where the actual withdrawals root value is not readily available.
 
+## Deposit Requests
+
+[EIP-6110] shifts deposit to the execution layer, introducing a new [EIP-7685] deposit request of type
```

Comment:
We should be very clear about the existence of the system contract bytecode. This means the deposit contract. Do we want to have the deposit contract now in the genesis state of the OP Stack as well as deployed as a system transaction? I don't think its really necessary to be honest

 We should also be very clear to say that DepositEvent logs emitted from the deposit contract are ignored. To be clear this is a deposit contract that is deployed to L2 rather than L1

Reply:
The deposit requests are populated from the events from the deposit contract

- It is underspecified if we include the deposit contract in the genesis state of the op stack, or if we deploy it via a network upgrade tx to chains upgrading

What are the pros and cons of including it or not including it?

Reply:
I guess the pros of shoving it in the genesis state are less of a diff to maintain since we'll need to increase the diff to deploy it via network upgrade txs.

Reply:
We always need to handle both the upgrade case and the from genesis case

Reply:
I think we should deploy these contracts similar to Holocene, otherwise the EL block processing code will have to differ more for Optimism. Right now, we still process these requests the same as L1, but just never send requests to the CL or include them in the block header.

Reply:
Actually, reading a bit more - seems like this just uses the existing deposit contract address. For this EIP, it should be a no-op. (Our contract won't emit any deposit events or include any requests in the block header or payload calls).

Reply:
Ah right believe @meyer9 is right here - this is what @emhane has done in `op-reth`, making it behave as a no-op.

Reply:
Going to land this pr now since we can follow up with changes if needed.